scripts/download_mmctl_release.sh

```diff
@@ -19,7 +19,7 @@ BIN_PATH=${2:-bin}
 THIS_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 if [[ "$THIS_BRANCH" =~ 'release-'[0-9] ]];
 then
-  RELEASE_TO_DOWNLOAD=$(echo $THIS_BRANCH | grep -Eo 'release-.*')
+  RELEASE_TO_DOWNLOAD=$(echo $THIS_BRANCH | grep -Eo 'release-([0-9](\.){0,1}).([0-9](\.){0,1})')
```

Comment:
```suggestion
  RELEASE_TO_DOWNLOAD=$(echo $THIS_BRANCH | grep -Eo 'release-([0-9](\.){0,1})\.([0-9](\.){0,1})')
```
Do we need to escape the dot in the middle ?

Reply:
It doesn't make a world of difference, but it is better escaping it. 👍 
Thanks for the comment @pfltdv 

Rebased and pushed the change